script.js

```diff
@@ -1,66 +1,73 @@
 const addAuthor = document.getElementById('Author');
 
-class Books {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
+function getlocalStorage() {
+  return JSON.parse(localStorage.getItem('Book Details'));
 }
 
-const storedBooks = [
-  //   {
-  //     title: 'Title 1',
-  //     author: 'Author 1'
-  //   },
-  //   {
-  //     title: 'Title 2',
-  //     author: 'Author 2'
-  //   }
-];
+function setlocalStorage(store) {
+  localStorage.setItem('Book Details', JSON.stringify(store));
+  console.log(store);
+}
+
+class BookDetails {
+  constructor(book = []) {
+    this.storedBooks = book;
+  }
+
+  addBook = function (book) {
+    const displayContainer = document.querySelector('.book-display-container');
+    const bookContainer = document.createElement('div');
+    bookContainer.innerHTML = `
+        <p class="book-title">${book.title}</p>
+        <p class="book-author">${book.author}</p>
+        <button class="remove-btn">remove</button>
+        <hr>
+    `;
+    displayContainer.appendChild(bookContainer);
+  }
 
-function addBook(book) {
-  const displayContainer = document.querySelector('.book-display-container');
-  displayContainer.innerHTML += `
-      <p class="book-title">${book.title}</p>
-      <p class="book-author">${book.author}</p>
-      <button class="remove-btn">remove</button>
-      <hr>
-  `;
+  removeBook = function (index, n = 1) {
+    this.storedBooks.splice(index, n);
+    setlocalStorage(this.storedBooks);
+  }
 }
 
+const store = new BookDetails();
+
 const addToBook = document.querySelector('.book-form');
 addToBook.addEventListener('submit', () => {
-//   e.preventDefault();
   // creat a new objet and add to it
   const title = addTitle.value;
   const author = addAuthor.value;
-
   const book = new Books(title, author);
-  storedBooks.push(book);
+  //  Push the book obj to array
+  store.storedBooks.push(book);
   // add book to List
-  addBook(book);
-  localStorage.setItem('Book Details', JSON.stringify(storedBooks));
+  store.addBook(book);
+  // store in local storage
+  setlocalStorage(store.storedBooks);
 });
 
 function load() {
-  const getbooks = JSON.parse(localStorage.getItem('Book Details'));
+  const getbooks = getlocalStorage();
   if (getbooks != null) {
     getbooks.forEach((bk) => {
-      addBook(bk);
-      storedBooks.push(bk);
+      store.addBook(bk);
+      store.storedBooks.push(bk);
     });
   }
 }
 
 // Load books
 load();
 
-const close = document.querySelectorAll('.remove-btn');
-close.forEach((ele, index) => {
-  ele.addEventListener('click', () => {
-    storedBooks.splice(index, 1);
-    localStorage.setItem('Book Details', JSON.stringify(storedBooks));
+const removeBooks = document.querySelectorAll('.remove-btn');
+removeBooks.forEach((book, index) => {
+  book.addEventListener('click', () => {
+    store.removeBook(index);
     setTimeout(window.location.reload(), 2000);
   });
```

Comment:
- [ ] You should not reload the page after adding or removing a book. Kindly render the books without reloading the page. :blush: